examples/tsconfig.json

```diff
@@ -7,13 +7,16 @@
         "inlineSourceMap": true,
         "importHelpers":true,
         "experimentalDecorators": true,
-        "noEmit": true,
+        "noEmit": false,
```

Comment:
> why did you change it to `true`

I didn't; I changed it to `false`, since it could have been a reason for the swallowed error messages. (But it didn't help.)